google/ggrc-core
# GGRC-5365 fixes hint to default assignees and verifiers

# Issue description

Reopen. Hint and valid values for Default Assignees*/Default Verifiers missed the Primary (Secondary) Contacts via previous fix in [GGRC-5365](https://github.com/google/ggrc-core/pull/9382)

This task is a blocker for [GGRC-7268](https://gojira.corp.google.com/browse/GGRC-7268)

# Steps to test the changes

1. Download assessment template import template
2. Check the hint for Default Assignees*/Default Verifiers

# Solution description

I just returned the values for the DEFAULT_PEOPLE_LABELS which were removed previously in this task

# Sanity checklist




# PR Review checklist
